src/lib/ollama.ts

```diff
@@ -42,6 +42,16 @@ export type OllamaTagResponse = {
 	models: OllamaModel[];
 };
 
+export async function ollamaTags() {
+	const settings = get(settingsStore);
+	if (!settings) throw new Error('No Ollama server specified');
+
+	const response = await fetch(`${settings.ollamaServer}/api/tags`);
+	if (!response.ok) throw new Error('Failed to fetch Ollama tags');
+
+	return response.json() as Promise<OllamaTagResponse>;
+}
```

Comment:
No changes needed here, but just so you know I will probably end up replacing all of these `fetch()` calls with Ollama's own JS library: https://github.com/ollama/ollama-js as part of https://github.com/fmaclen/hollama/issues/103#issuecomment-2253706258